content/docs/portals.md

```diff
@@ -27,34 +28,34 @@ render() {
 }
 ```
 
-However, sometimes it's useful to insert a child into a different location in the DOM:
+Sin embargo, a veces es útil insertar un hijo en una ubicación diferente en el DOM:
 
 ```js{6}
 render() {
-  // React does *not* create a new div. It renders the children into `domNode`.
-  // `domNode` is any valid DOM node, regardless of its location in the DOM.
+  // React *no* crea a nuevo div, convierte el hijo en `domNode`.
+  // `domNode` es cualquier nodo DOM válido, independientemente de su ubicación en el DOM.
   return ReactDOM.createPortal(
     this.props.children,
     domNode
   );
 }
 ```
 
-A typical use case for portals is when a parent component has an `overflow: hidden` or `z-index` style, but you need the child to visually "break out" of its container. For example, dialogs, hovercards, and tooltips.
+Un caso de uso típico de los portales es cuando un componente principal tiene un estilo `overflow: hidden` or `z-index`, pero necesita que el elemento "salga" visualmente de su contenedor. Por ejemplo, cuadros de diálogo, hovercards y tooltips.
```

Comment:
- "Portals" instead of "los portales" (?
- "componente padre" instead of "componente principal"

Reply:
@jonalvarezz  It is necessary to keep the same style in this word throughout the document, we speak of `principal` or `padre`. I like more to always keep for all the meaning `padre`